cts/CTStests.py

```diff
@@ -1693,6 +1693,19 @@ def __init__(self, cm):
         self.stopall = SimulStopLite(cm)
         self.is_unsafe = 0 # Handled by canrunnow()
 
+    def _is_managed(self, node):
+        is_managed = self.rsh(node, "crm_attribute -t rsc_defaults -n is-managed -Q -G -d true", 1)
+        is_managed = is_managed[:-1] # Strip off the newline
+        return is_managed == "true"
+
+    def _set_unmanaged(self, node):
+        self.debug("Disable resource management")
+        self.rsh(node, "crm_attribute -t rsc_defaults -n is-managed -v false")
+
+    def _set_managed(self, node):
+        self.debug("Re-enable resource management")
+        self.rsh(node, "crm_attribute -t rsc_defaults -n is-managed -D")
+
```

Comment:
ahhahaha.  we came up with the same commits.
did you ever figure out why the old code stopped working? (i tried, failed, fixed it in much the same way you did)